src/sentry/static/sentry/app/views/releases/detail/overview/chart/healthChart.tsx

```diff
@@ -130,7 +130,7 @@ class HealthChart extends React.Component<Props> {
 
     if (timeseriesData.every(s => s.data.length === 1)) {
       if (zoomRenderProps.period) {
-        const {start, end} = parseStatsPeriod(zoomRenderProps.period, null);
+        const {start, end} = parseStatsPeriod(zoomRenderProps.period);
```

Comment:
The null here is important, we want to explicitly pass it into the `moment.format`.

Otherwise, it will be undefined and will be replaced by the default value (DATE_TIME_FORMAT)

Reply:
good catch Matej!